contracts/zerodev/account/kernel/WalletKernel.sol

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@gnosis.pm/safe-contracts/contracts/GnosisSafe.sol";
+import "@openzeppelin/contracts/utils/cryptography/draft-EIP712.sol";
+import "../../plugin/IPlugin.sol";
+import "../../../core/Helpers.sol";
+import "../../../interfaces/IAccount.sol";
+import "../../utils/OpinionatedExec.sol";
+import "./Compatibility.sol";
+
+struct WalletKernelStorage {
+    address owner;
+    uint256 nonce;
+}
+
+
+/// @title Wallet Kernel
+/// @author taek<leekt216@gmail.com>
+/// @notice wallet kernel for minimal wallet functionality
+/// @dev supports only 1 owner and 1 threshold, multiple plugins
+contract WalletKernel is IAccount, EIP712, Compatibility {
```

Comment:
I suggest that we rename this to `Kernel` cuz first of all it's more of an account kernel rather than wallet, but secondly it's easier to say "Kernel vs Safe"

Reply:
We want to argue that "kernel is better than safe for the next generation of AA-native wallets"

Reply:
Cool 👍

Reply:
Actually it looks like it's just called [Safe](https://github.com/safe-global/safe-contracts/blob/main/contracts/Safe.sol#L33) now so maybe let's just rename this to `Kernel`.